presto-docs/src/main/sphinx/develop/serialized-page.rst

```diff
@@ -212,6 +212,20 @@ from the null flags.
 .. image:: serialized-page-row-column.png
   :width: 600
 
+DICTIONARY Encoding
+-------------------
+
+* Number of rows - 4 bytes
+* Dictionary values column
```

Comment:
Correct me if I'm wrong, but I believe this may be a nested dictionary.  At the very least, should we consider mentioning that this values type is recursive (i.e. could be other types listed in this doc)?

Reply:
I believe this is so. It would be similar to how 'elements' column of the ARRAY can have any encoding. 

Do you have any suggestion about where / how to mention this?

Reply:
Perhaps something like this: `Dictionary values column.  This is column is itself a serialized block, meaning its format may be that of any type mentioning in this document.`

Reply:
@tdcmeehan Thank you, Tim. Updated. Would you take another look?